package.json

```diff
@@ -38,7 +39,7 @@
     "@visx/shape": "2.12.2",
     "@visx/text": "1.7.0",
     "@visx/tooltip": "^2.10.0",
-    "ace-builds": "^1.4.13",
+    "ace-builds": "1.4.14",
```

Comment:
I think we're limited to 1.4.X because of react-ace. I'm just pinning this here to see if I can fix some e2e tests. Any real upgrade should happen in another PR.